api/views.py

```diff
@@ -45,6 +47,15 @@ class ExportInstructorLocationsView(ListAPIView):
     serializer_class = ExportInstructorLocationsSerializer
 
 
+class ExportMembersView(ListAPIView):
+    """Show everyone who qualifies as an SCF member."""
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    paginator = None  # disable pagination
+
+    queryset = get_members()
```

Comment:
Will that line trigger evaluation of `get_queryset` once we implement 
it?  And are the querysets for other things like `ExportBadgesView` also 
being triggered on every HTTP request?